dev_menu.py

```diff
@@ -128,6 +151,9 @@ def clip(x, mini, maxi):
 
 @retry((ValueError, RuntimeError), 3, delay_s = 0)
 def show_menu(items: List[str], header=None) -> int:
+    print()
+    print('-- MXNet dev menu --')
+    print()
```

Comment:
Nit: this would look better to me in a single line
```suggestion
    print('\n-- MXNet dev menu --\n')
```